lib/weapon.rb

```diff
@@ -0,0 +1,27 @@
+class Weapon
+
```

Comment:
Maybe this class should be named something else - it doesn't represent a "weapon" in itself, but its responsibility seems to be computing the winning move. So maybe something like `GameResult` would be clearer?

In case we make this change, then it's probably not needed anymore to store instances in the Game's `@weapon` and `@computer_weapon`, as those can be symbol/string values, and we could only instantiate `GameResult` at the time when we need to calculate the result